sentry-apache-http-client-5/src/main/java/io/sentry/transport/apache/ApacheHttpClientTransport.java

```diff
@@ -129,6 +132,19 @@ public void cancelled() {
     }
   }
 
+  @Override
+  public void flush(long timeoutMillis) {
+    try {
+      boolean b = currentlyRunning.waitTillZero(timeoutMillis, TimeUnit.MILLISECONDS);
```

Comment:
let's just name things accordingly, maybe `result` would be already better

Reply:
yeah, this sneaked in.